schema.sql

```diff
@@ -0,0 +1,63 @@
+DROP DATABASE IF EXISTS books;
+
+CREATE DATABASE books;
+
+USE books;
+
+CREATE TABLE bookInfo (
+  id int NOT NULL AUTO_INCREMENT,
+  title VARCHAR(100) NOT NULL,
+  author VARCHAR(100) NOT NULL,
+  description TEXT NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE image (
+  id int NOT NULL AUTO_INCREMENT,
+  bookInfo_id int,
+  image BLOB,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT,
+  email VARCHAR(50) NOT NULL,
+  PRIMARY KEY (id)
+);
+
+
+CREATE TABLE ratings (
+  id INT NOT NULL AUTO_INCREMENT,
+  bookInfo_id int,
+  user_id int,
+  rating TINYINT,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE reviews (
+  id INT NOT NULL AUTO_INCREMENT,
+  ratings_id int,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE readStatus (
+  id INT NOT NULL AUTO_INCREMENT,
+  bookInfo_id int,
+  user_id int,
+  status TINYINT,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE shelf (
```

Comment:
I'm not sure what data is contained within shelf and bookshelf

Reply:
The relationship between shelf and book is many to many. And this is reflected as bookShelf table.

But I realized I need to remove user_id from bookShelf table and add it to shelf table.